src/index.jsx

```diff
@@ -117,7 +117,7 @@ const getDefaultIcon = () => {
 
 const getAppNamePrefix = () => {
   const appNode = document.querySelector(APP_SELECTOR)
-  return appNode.dataset.cozyAppNamePrefix || appNode.dataset.cozyAppEditor || undefined
+  return appNode.dataset.cozyAppNamePrefix || undefined
```

Comment:
It's a bad pattern to return `undefined`, as `undefined` means that a variable did not get value assigned. Instead, we should return `null` or an empty string `''`.